### PR TITLE
Spelling corrections in comments/javadocs

### PR DESCRIPTION
**Pull request by cjmctague**

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/client/admin/ActiveCompaction.java [hunk 1]
```diff
@@ -77,7 +77,7 @@
   public abstract String getTable() throws TableNotFoundException;
 
   /**
-   * @return tablet thats is compacting
+   * @return tablet that's is compacting
```
Comment (milleruntime) [inline core/src/main/java/org/apache/accumulo/core/client/admin/ActiveCompaction.java]: Should  be "that is" or "that's"

File: core/src/main/java/org/apache/accumulo/core/client/sample/Sampler.java [hunk 2]
```diff
@@ -56,7 +56,7 @@
   /**
    * @param k
    *          A key that was written to a rfile.
-   * @return True if the key (and its associtated value) should be stored in the rfile's sample.
+   * @return True if the key (and its associated value) should be stored in the rfiles sample.
```
Comment (milleruntime) [inline core/src/main/java/org/apache/accumulo/core/client/sample/Sampler.java]: The apostrophe for rfile is showing possession here so should be "rfile's"

File: server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java [hunk 3]
```diff
@@ -1535,7 +1535,8 @@ public TConditionalSession startConditionalUpdate(TInfo tinfo, TCredentials cred
         try {
           TabletServer.this.resourceManager.waitUntilCommitsAreEnabled();
         } catch (HoldTimeoutException hte) {
-          // Assumption is that the client has timed out and is gone. If thats not the case throw an
+          // Assumption is that the client has timed out and is gone. If that's not the case throw
+          // an
```
Comment (milleruntime) [inline server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java]: Could combine this line with the next line.

Comment (cjmctague) [reply server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java]: I don't recall changing that. Might have been the formatter. I'll move the an to the next line

File: core/src/main/java/org/apache/accumulo/core/client/sample/Sampler.java (hunk 2 above)
Comment (cjmctague) [reply core/src/main/java/org/apache/accumulo/core/client/sample/Sampler.java]: Fixed

File: core/src/main/java/org/apache/accumulo/core/client/admin/ActiveCompaction.java (hunk 1 above)
Comment (cjmctague) [reply core/src/main/java/org/apache/accumulo/core/client/admin/ActiveCompaction.java]: Fixed

**Comment by milleruntime** [comment]
If you'd like to be added to the contributors page at https://accumulo.apache.org/people , please submit a pull request to update the website page at https://github.com/apache/accumulo-website/blob/master/pages/people.md with your details (please mention #620 )

File: server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java (hunk 3 above)
Comment (milleruntime) [reply server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java]: It was probably from a recent commit. Thanks for the fix!

**Pull request merged**